interaction-manager/src/lib_components/spatial-navigation.js

```diff
@@ -206,19 +207,29 @@ class SpatialNavigation {
     moveFocus(direction) {
         if (!this.enabled) return;
 
-        const focusableGroup = this.getFocusableGroup();
-        const { x, y, width, height } = document.activeElement.getBoundingClientRect();
+        const activeElement = this.isActiveElementInGroup(document.activeElement) ?
+            document.activeElement : this.lastFocusedElement;

```

Comment:
Instead of doing this long check, can we make a function that is `checkActiveElementInGroup` which returns the correct element?